spec/cars_controller_spec.rb

```diff
@@ -0,0 +1,87 @@
+require 'spec_helper'
+
+describe "Cars Controller" do
+  let!(:user_example) {User.create(
```

Comment:
You could probably save a few keystrokes and call this :user. It's generally assumed that it'll be an example since these are tests.